datamodels.py

```diff
@@ -622,6 +632,9 @@ def get_public_courses():
 	session = get_session()
 	return session.query(Course).filter(Course.guest_access == True).all()
 
+def get_program_by_slug(slug):
```

Comment:
This can stay, but the real reason we're maintaining these definitions is to maintain backwards compatibility from an earlier version.  In the future, we should be migrating to class or static methods.

Reply:
Ah, that wasn't clear for me and wondering why it was done this way. Thanks!